Makefile

```diff
@@ -25,11 +29,18 @@ pack:
 
 release: build pack
 
-bindata: 
-	go-bindata -o=modules/bindata/bindata.go -ignore="\\.DS_Store|README.md" -pkg=bindata conf/...
+bindata: modules/bindata/bindata.go
+
+modules/bindata/bindata.go: $(DATA_FILES)
```

Comment:
Same as below, why not just use one? Is this some high-tech on make command (very little knowledge on makefile)?